recipes-android/brcm-patchram-plus/brcm-patchram-plus_git.bbappend

```diff
@@ -1,3 +1,3 @@
-FILESEXTRAPATHS_prepend_sawfish := "${THISDIR}/brcm-patchram-plus:"
-SRC_URI_append_sawfish = " file://patchram.service "
-CFLAGS_append_sawfish = " -DLPM_SAWSHARK"
+FILESEXTRAPATHS:prepend:ray := "${THISDIR}/brcm-patchram-plus:"
+SRC_URI:append:ray = " file://patchram.service "
+CFLAGS:append:ray = " -DLPM_SAWSHARK"
```

Comment:
are you sure about `-DLPM_SAWSHARK`?

Reply:
Bluetooth works differently on the Skagen Falster 2 (uses hci-smd, similar to `swift`).
It's not functional for the moment anyway.
Will remove the recipe for now.